EVSVesuvio/scripts/__init__.py

```diff
@@ -31,15 +32,22 @@ def __set_up_parser():
 def __setup_config(args):
     config_dir = handle_config.VESUVIO_CONFIG_PATH
     handle_config.setup_config_dir(config_dir)
+    ipfolder_dir = handle_config.VESUVIO_IPFOLDER_PATH
 
     if handle_config.config_set():
         cache_dir = handle_config.read_config_var('caching.location') if not args or not args.set_cache else args.set_cache
         experiment = handle_config.read_config_var('caching.experiment') if not args or not args.set_experiment else args.set_experiment
+        ipfolder_dir = handle_config.read_config_var('caching.ipfolder') if not args or not args.set_ipfolder else args.set_ipfolder
     else:
         cache_dir = config_dir if not args or not args.set_cache else args.set_cache
         experiment = "default" if not args or not args.set_experiment else args.set_experiment
+        ipfolder_dir = ipfolder_dir if not args or not args.set_ipfolder else args.set_ipfolder
+
+        handle_config.setup_default_ipfile_dir()
+
     handle_config.set_config_vars({'caching.location': cache_dir,
-                                   'caching.experiment': experiment})
+                                   'caching.experiment': experiment,
+                                   'caching.ipfolder': ipfolder_dir})
```

Comment:
So I got a bit carried away and ended up changing other parts of the code as well. As it is now, it will warn the user when the IP file is not found, and if the user tries to run the script, it does (eventually) fail.